README.adoc

```diff
@@ -84,12 +85,11 @@ public class SuccessfulCheck implements HealthCheck {
 }
 ```
 
-### Integration with CDI
+=== Integration with CDI
 
-Within CDI contexts, beans that implement `HealthCheck` and annotated with `@Health` are discovered automatically and are invoked by the framework or runtime when the outermost protocol entry point (i.e. `http://HOST:PORT/health`) receives an inbound request.
+Within CDI contexts, beans that implement `HealthCheck` are discovered automatically and are invoked by the framework or runtime when the outermost protocol entry point (i.e. `http://HOST:PORT/health`) receives an inbound request.
```

Comment:
The previous sentence asks to annotated with @Health. You should not change this under this issue though. We can discuss this under a different issue on whether it is necessary to require `@Health`. I see there is no point either. However, we need to discuss this under a different issue.

Reply:
This is more a cleanup as `@Health` has been deprecated in the last version of the spec. I will revert this commit.